db/migrate/20250117163617_add_changed_fields_to_content_block_versions.rb

```diff
@@ -0,0 +1,13 @@
+class AddChangedFieldsToContentBlockVersions < ActiveRecord::Migration[7.1]
+  def up
+    change_table :content_block_versions, bulk: true do |t|
+      t.json "changed_fields"
```

Comment:
I put this as a json blob first because I wasn't sure what structure it could have or if it would be different for different versions... but then I formalised it as a list of  a Data type `ChangedFields`. If we're happy with this structure I'm thinking I should create a table of ChangedFields with a many to one relationship with Versions, and then the data is enshrined in the db and will be easier to create etc. But then again it would mean any changes (e.g. if the design of the UI changes) might be harder to make 🤔

Reply:
I do wonder what the benefit of saving this stuff would be. I don't think it'll be a massive strain on the DB to do it at load time (unless, of course, the history gets very long), but then that might be time to introduce pagination anyway.